components/UI/modalProfilePic.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from "react";
+import styles from "../../styles/components/modalMessage.module.css";
+import ppStyles from "../../styles/components/profilePic.module.css";
+import { FunctionComponent } from "react";
+import { Modal } from "@mui/material";
+import ClickableAction from "./iconsComponents/clickableAction";
+import theme from "../../styles/theme";
+import DoneFilledIcon from "./iconsComponents/icons/doneFilledIcon";
+import ArrowLeftIcon from "./iconsComponents/icons/arrowLeftIcon";
+import { useContractWrite, useTransactionManager } from "@starknet-react/core";
+import { Call } from "starknet";
+import identityChangeCalls from "../../utils/callData/identityChangeCalls";
+import { hexToDecimal } from "../../utils/feltService";
+import { getImgUrl } from "../../utils/stringService";
+
+type ModalProfilePicProps = {
+  closeModal: (cancel: boolean) => void;
+  open: boolean;
+  nft: StarkscanNftProps;
+  id: string;
+  setPfpTxHash: (hash: string) => void;
```

Comment:
- `nft` is not a good naming imo because Idk
- `open` ==> `isModalOpen`
- `id` What id is it ? I think you can also make a better naming